ush/python/pygfs/task/atmens_analysis.py

```diff
@@ -142,6 +143,21 @@ def initialize_analysis(self) -> None:
         FileHandler(bias_dict).sync()
         logger.debug(f"Bias correction files:\n{pformat(bias_dict)}")
 
+        # Extract radiance bias correction files from tarball
+        for action, filelist in bias_dict.items():
+            if 'copy' in action:
+                for sublist in filelist:
+                    if len(sublist) != 2:
+                        raise Exception(
```

Comment:
```suggestion
                        logger.exception("FATAL ERROR: Expected list of incorrect length")
                        raise IndexError(
```
The list is of incorrect length, so indexing anything other than 1, 2 will raise this error.

Reply:
This section of `atmens_analysis.py` has been changed.  [Click here](https://github.com/RussTreadon-NOAA/global-workflow/blob/bdffc0217359068df414e72ed6a03a203d68ad4e/ush/python/pygfs/task/atmens_analysis.py#L139-#L150) for the current script , echoed below for convenience
```
        # stage bias corrections
        logger.info(f"Staging list of bias correction files generated from JEDI config")
        self.task_config.VarBcDir = f"{self.task_config.COM_ATMOS_ANALYSIS_PREV}"
        bias_file = f"rad_varbc_params.tar"
        bias_dict = self.jedi.get_bias_dict(self.task_config, bias_file)
        FileHandler(bias_dict).sync()
        logger.debug(f"Bias correction files:\n{pformat(bias_dict)}")

        # extract bias corrections
        logger.info(f"Extract bias correction files from tarball")
        self.jedi.extract_tar(self.task_config, bias_dict)
```